### PR TITLE
Create compatibility_table

### PR DESCRIPTION
**Pull request by supernova-at**
Creates first suite compatibility table.
This table should be updated with every release.

Please update and move to proper final location.

I would not merge this until after the GA release is complete on Friday, December 13th.

### CODE REVIEW

**Comment by jeff-matthews** [comment]
Thanks @supernova-at 🙏 

We'll find a good place for this for GA.

**Comment by jeff-matthews** [comment]
I made some adjustments using a vanilla .md table. I think this is sufficient for GA, but we should revisit and consider creating a separate component for the compatibility table as a fast follow. As it grows, it'll be harder to use if the columns or headers aren't sticky.

We should also link to the GitHub changelog from this page when it's ready for 1.0.0.

Screenshot 2024-12-09 at 3 05 55 PM

**Comment by jeff-matthews** [comment]
Converting PR to draft so we don't merge before GA.

File: src/content/docs/get-started/release.mdx
Comment (keharper) [inline src/content/docs/get-started/release.mdx]:
```suggestion
## Compatibility table
```

File: src/content/docs/get-started/release.mdx
Comment (keharper) [inline src/content/docs/get-started/release.mdx]:
```suggestion
Adobe organizes storefront application releases into logical suite versions and follow a regular release schedule (approximately once a month). However, Adobe may release updates as necessary.
```

**Review by keharper — APPROVED** [review]

File: src/content/docs/get-started/release.mdx
Comment (supernova-at) [inline src/content/docs/get-started/release.mdx:37]: @fnhipster @sirugh @shiftedreality please review this section with an eye on:

1. Should we include functions coming from the SDK at this point? The SDK is not officially `1.0.0` yet but breaking the signatures of something like `initialize` or `render` will break customer sites. If we do include these functions, please enumerate the "public" ones we expect customers to use (some question marks in the following lines)
2. Anything that I missed?

File: src/content/docs/get-started/release.mdx
Comment (supernova-at) [inline src/content/docs/get-started/release.mdx]: We probably don't want to actually link to this development documentation site. Not sure how best to enumerate all of these tokens otherwise.

File: src/content/docs/get-started/release.mdx
Comment (supernova-at) [inline src/content/docs/get-started/release.mdx]: Are there functions from the `event-bus` that we should include?

File: src/content/docs/get-started/release.mdx
Comment (supernova-at) [inline src/content/docs/get-started/release.mdx]: I listed at least `render` and `initialize` here ... are there others?

File: src/content/docs/get-started/release.mdx
Comment (jeff-matthews) [reply src/content/docs/get-started/release.mdx]: We might be able to pull in a subset of the SDK docs via component that @bdenham developed in this repo: https://github.com/commerce-docs/dropins.dev

Comment (jeff-matthews) [reply src/content/docs/get-started/release.mdx]: Actually, can we just link here @supernova-at?

https://experienceleague.adobe.com/developer/commerce/storefront/dropins/all/branding/#design-tokens

Comment (supernova-at) [reply src/content/docs/get-started/release.mdx]: Yes, perfect!

File: src/content/docs/get-started/release.mdx
Comment (sirugh) [inline src/content/docs/get-started/release.mdx:16]: Where is the version noted in code? As tag in github on boilerplate repo? Just in the docs here? There's a package.json in boilerplate, so I guess we could bump that.

Infact if we want to do that, we need it as a step during release to commit the version bump to develop before merging

File: src/content/docs/get-started/release.mdx
Comment (sirugh) [inline src/content/docs/get-started/release.mdx]: Is this the agreed on flow? Will we continue using `develop` past GA and merge into main once a month for these "suite releases"?

File: src/content/docs/get-started/release.mdx
Comment (sirugh) [inline src/content/docs/get-started/release.mdx]: So...when we release a dropin and update `main` on boilerplate, should we expect to bump the suite version?

File: src/content/docs/get-started/release.mdx
Comment (sirugh) [reply src/content/docs/get-started/release.mdx:37]: I don't actually think we should include SDK in docs. It is not public. what is public is the _output_ packages of the SDK, including fetchGql, tools, event-bus, etc.

But I do think we should, like you say, include signatures of render/initialize/etc. those come from tools

File: src/content/docs/get-started/release.mdx
Comment (sirugh) [reply src/content/docs/get-started/release.mdx]: The event bus has `on` and `emit`.

File: src/content/docs/get-started/release.mdx
Comment (sirugh) [inline src/content/docs/get-started/release.mdx:21]: In terms of "suite", and what we include here, I think you can just check the boilerplate package.json and list them here, including "@dropins/build-tools", @dropins/tools", for example.

https://github.com/hlxsites/aem-boilerplate-commerce/blob/develop/package.json#L28

File: src/content/docs/get-started/release.mdx
Comment (supernova-at) [reply src/content/docs/get-started/release.mdx]: No, not always. Package versions yes, and we'll update the package ranges

File: src/content/docs/get-started/release.mdx
Comment (supernova-at) [reply src/content/docs/get-started/release.mdx:16]: Discussion in https://github.com/commerce-docs/microsite-commerce-storefront/pull/141#discussion_r1882475358.

File: src/content/docs/get-started/release.mdx
Comment (fnhipster) [reply src/content/docs/get-started/release.mdx]: Is this about the Boilerplate? Each drop-in has its release strategy in place.

Comment (supernova-at) [reply src/content/docs/get-started/release.mdx]: Yes (doesn't necessarily have to be `develop`, but some branch, yeah). More details here: https://wiki.corp.adobe.com/display/COREENG/Post-GA+Drop-ins+Release+Plan.

File: src/content/docs/get-started/release.mdx
Comment (fnhipster) [reply src/content/docs/get-started/release.mdx]: and `lastPaylod` https://git.corp.adobe.com/Commerce-Storefront/StorefrontSDK/blob/434bb967125b90ac02219a76a7def43fcc0d5b62/packages/event-bus/src/index.ts#L31

File: src/content/docs/get-started/release.mdx
Comment (fnhipster) [reply src/content/docs/get-started/release.mdx]: You can review the files exposed by `@dropins/tools` at https://www.npmjs.com/package/@dropins/tools?activeTab=code

`types/signatures are in /@dropins/tools/types/elsie/src/`

File: src/content/docs/get-started/release.mdx
Comment (supernova-at) [reply src/content/docs/get-started/release.mdx:21]: Fixed in https://github.com/commerce-docs/microsite-commerce-storefront/pull/141/commits/3b8df5cdcdd19a3ae0fa52faf2039669bdddb4b0

File: src/content/docs/get-started/release.mdx
Comment (supernova-at) [reply src/content/docs/get-started/release.mdx]: Can you help me word this one @fnhipster ? I don't want to list everything there.

**Comment by supernova-at** [comment]
I'm done making changes to this one, @jeff-matthews

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**
